Controller/ExceptionController.php

```diff
@@ -71,68 +53,17 @@ protected function getViewHandler()
      */
     public function showAction(Request $request, FlattenException $exception, DebugLoggerInterface $logger = null)
     {
-        try {
-            $format = $this->getFormat($request, $request->getRequestFormat());
-        } catch (\Exception $e) {
-            $format = null;
-        }
-        if (null === $format) {
-            $message = 'No matching accepted Response format could be determined, while handling: ';
-            $message .= $this->getExceptionMessage($exception);
-
-            return $this->createPlainResponse($message, Response::HTTP_NOT_ACCEPTABLE, $exception->getHeaders());
-        }
-
-        $currentContent = $this->getAndCleanOutputBuffering(
-            $request->headers->get('X-Php-Ob-Level', -1)
-        );
+        $currentContent = $this->getAndCleanOutputBuffering($request->headers->get('X-Php-Ob-Level', -1));
         $code = $this->getStatusCode($exception);
-        $parameters = $this->getParameters($currentContent, $code, $exception, $logger, $format);
-        $showException = $request->attributes->get('showException', $this->showException);
```

Comment:
Yeah the request attributes are reinitialized in the ExceptionListener so this is useless IMO.